src/commands/install/utils/extract_info_from_repo_id.py

```diff
@@ -0,0 +1,90 @@
+import re
+from typing import NamedTuple, Optional
+
+from .. import installation_exceptions
```

Comment:
Relative imports are inadvisable in python, for `__init__`s they are fine but i'd rather have an explicit import in the implementation, when repo happens to change its structure somehow

Reply:
By `explicit` do you mean `absolute` imports? 
Why are relative imports inadvisable in Python? Usually, I choose relative or absolute imports based on whichever gives fewer jumps between directories.

Reply:
Yep that's what i meant